src/nunit.analyzers.playground/Program.cs

```diff
@@ -27,9 +27,14 @@ static void Main(string[] args)
             Program.Foo(22);
         }
 
-        public static void Foo(short a) { }
+        public static void Foo(short a)
+        {
+        }
 
         [TestCase(3d, "a value", 123, ExpectedResult = 22)]
-        public int RunTest(double a, string b, int c) { return 22; }
+        public int RunTest(double a, string b, int c)
+        {
+            return 22;
+        }
```

Comment:
This is fine. I think that we at some point will remove the "playground" from the project, but don't spend time reverting this 😄 .

Reply:
Sorry about this; could we maybe get an .editorconfig in here to fix this up? My workflow totally hosed up a bunch of these files (sorry!)

Reply:
Don't be sorry. It is more my fault than yours. There is a .editorconfig in the project, but I think that I left the playground project and the integrationtests project out in my cleanup as I planned to remove them at some time, as I don't think they provide much value (and actually the integrationtests project is not being build at the moment).